### PR TITLE
Update version after Feb release.

### PR DESCRIPTION
**Pull request by chenrujun**
1. Update version after Feb release.
2. Fix link error in README.md.

### CODE REVIEW

File: appconfiguration/feature-management-sample/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Feature Management Sample shared library for Java
 
-This sample describes how to use [spring-cloud-feature-management](https://github.com/Azure/azure-sdk-for-java/blob/azure-spring-boot_3.6.0/sdk/appconfiguration/spring-cloud-azure-feature-management/README.md) to manage features and how to get configurations from Azure Configuration Service to Spring Environment.
+This sample describes how to use [spring-cloud-azure-feature-management](https://github.com/Azure/azure-sdk-for-java/blob/spring-cloud-azure-feature-management_1.3.0/sdk/appconfiguration/spring-cloud-azure-feature-management/README.md) to manage features and how to get configurations from Azure Configuration Service to Spring Environment.
```
Comment (chenrujun) [inline appconfiguration/feature-management-sample/README.md:3]: FYI: Fix error here.

File: appconfiguration/feature-management-web-sample/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Feature Management Web Sample shared library for Java
 
-This sample describes how to use [spring-cloud-feature-management](https://github.com/Azure/azure-sdk-for-java/blob/azure-spring-boot_3.6.0/sdk/appconfiguration/spring-cloud-azure-feature-management/README.md) to manage features and how to get configurations from Azure Configuration Service to Spring Environment.
+This sample describes how to use [spring-cloud-azure-feature-management](https://github.com/Azure/azure-sdk-for-java/blob/spring-cloud-azure-feature-management_1.3.0/sdk/appconfiguration/spring-cloud-azure-feature-management/README.md) to manage features and how to get configurations from Azure Configuration Service to Spring Environment.
```
Comment (chenrujun) [inline appconfiguration/feature-management-web-sample/README.md:3]: FYI: Fix error here.

File: keyvault/azure-securtiy-keyvault-jca/run-with-command-line-client-side/README.md [hunk 3]
```diff
@@ -49,15 +49,15 @@ jdk 11.0.12 or above
    ```
 1. Get the azure-security-keyvault-jca.jar. You can download the latest published jar from maven
    repository [azure-security-keyvault-jca][azure-security-keyvault-jca]. When this document is 
-   written, the latest jar is azure-security-keyvault-jca.2.1.0.jar
+   written, the latest jar is azure-security-keyvault-jca.2.6.0.jar
```
Comment (chenrujun) [inline keyvault/azure-securtiy-keyvault-jca/run-with-command-line-client-side/README.md:52]: FYI: I also updated version for jca here.

**Review by backwind1233 — APPROVED** [review]
LGTM

**Pull request merged**